internal/host/freeaddressescmd.go

```diff
@@ -64,18 +66,33 @@ func (f *freeAddressesCmd) GetSteps(ctx context.Context, host *models.Host) ([]*
 	if err != nil {
 		return nil, err
 	}
+
+	const containerName = "free_addresses_scanner"
+
+	podmanRunCmd := strings.Join([]string{
+		"run", "--privileged", "--net=host", "--rm", "--quiet",
+		"--name", containerName,
+		"-v", "/var/log:/var/log",
+		"-v", "/run/systemd/journal/socket:/run/systemd/journal/socket",
+		f.freeAddressesImage,
+		"free_addresses",
+		param,
+	}, " ")
+
+	// Sometimes the address scanning takes longer than the interval we wait between invocations.
+	// To avoid flooding the log with "container already exists" errors, we silently fail by manually
+	// checking if it exists and only running if it doesn't
+	checkAlreadyRunningCmd := fmt.Sprintf("podman ps --format '{{.Image}}' | grep -q '^%s$'", containerName)
```

Comment:
podman ps --format '{{.Image}}' provides the image name.  Since we are using the same image for all tools, it is incorrect.  podman ps --format '{{.Names}}' will be more appropriate

Reply:
> podman ps --format '{{.Image}}' provides the image name. Since we are using the same image for all tools, it is incorrect. podman ps --format '{{.Names}}' will be more appropriate

Right, meant to get the container's name. It's weird that it didn't cause any problems in my testing.